vertx-core/src/main/asciidoc/http.adoc

```diff
@@ -383,7 +383,7 @@ Here's an example of querying and adding cookies:
 
 ==== Handling compressed body
 
-Vert.x can handle compressed body payloads which are encoded by the client with the _deflate_, _gzip_ or _brotli_
+Vert.x can handle compressed body payloads which are encoded by the client with the _deflate_, _gzip_, _snappy_ or _brotli_
```

Comment:
I think we should update the doc and mention that snappy does not require an external dependency unlike brotli

Reply:
that would be great to improve the documentation @lucamolteni because this section is rather slim, please go ahead

Reply:
@vietj can I do in another PR though? Let's merge this first

Reply:
actually I think it would be better to have this done in the same PR , since we will need to backport to 4.x branches

Reply:
@vietj I've updated the two sections accordingly, please take a look

Reply:
Any news @vietj ? I'm closing this